### ISSUE TITLE
Enhance 'flooding' Property to Include Detailed Flood Risk Information

### ISSUE THREAD

**Issue by mir491** [Property-Data-Trust-Framework/schemas#112]
I propose enhancing the 'flooding' property in the PDTF specification to include more detailed flood risk information. The current structure allows for indicating whether a flood risk report has been prepared and whether any part of the property has been flooded. However, it lacks the ability to provide detailed flood risk assessments and related data.
```
"floodRisk": {
                      "title": "Flood Risk Information",
                      "type": "object",
                      "properties": {
                          "risk": {
                              "title": "Flood Risk Level",
                              "type": "string",
                              "enum": [
                                  "Very Low",
                                  "Low",
                                  "Moderate",
                                  "High",
                                  "Very High"
                              ]
                          }
                    }
```

**Comment by edmolyneux** [comment]
Detailed flood risk reports form part of the searches under the Environmental content type.

**Comment by edmolyneux** [comment]
The 'flood risk report' is part of the TA6 so has to be there and is arguably a duplication - once the enviro report returns we might choose to set that to "Yes" as well...

**Comment by mir491** [comment]
yes under environmentalRisk we do have riskResults but we want to provide atleast some sort of risk information under property details itself. may be we can remove details and just put flood risk level

**Comment by edmolyneux** [comment]
Well we shouldn't invent a new structure for those results, since only the results as returned from the search provider will be provenanced. So we need to duplicate the entire flood report here.

**Comment by mir491** [comment]
yes that is correct

**Comment by edmolyneux** [comment]
We did have a group conversation a while back about moving the search results into the body of the material info, and agreed not to. Should we revisit?

**Comment by mir491** [comment]
yes i think, we need to have some flood risk info in property details as most of the providers like sprift, street etc are providing flood risk as part of property details

**Comment by edmolyneux** [comment]
We do have a schema working group catchup tomorrow at 12pm UK-time - @vivek656 let's raise this there shall we?

**Comment by edmolyneux** [comment]
@vivek656 thinking about this some more last night and I'm coming round to the idea for sure. We should reconsider our 'searches are separate' thinking and move LLC, CON29R and DW in to the appropriate planning and water/drainage areas too. Let's confirm everyone's happy with this today and then I'd be happy to address this one 👍

**Comment by vivek656** [comment]
@edmolyneux , me and others discussed this in biweekly meetings. and @edmolyneux will transfer(copy) `results` information from the searches section to material info. then we can add this data to the specific flooding risk report field.
@edmolyneux please, correct and add if I missed something

**Comment by edmolyneux** [comment]
Will be addressed as part of #88
